spec/models/project_spec.rb

```diff
@@ -213,27 +213,45 @@
       context "recently created" do
         let!(:project) { create(:project, platform: "Go", name: "github.com/some-nonexistent-fake/pkg", status: nil, created_at: 1.hour.ago) }
 
-        it "should not mark it as Removed" do
+        it "should not mark it as Removed for a 404" do
```

Comment:
🎱 Thoughts about cleaning up the specs to standardize the mocking / separate out the setup from the test expectations?